docs/content/docs/install/manual_demo/_index.md

```diff
@@ -24,20 +24,40 @@ The OSM Manual Install Demo Guide is a step by step set of instructions to quick
 
 Use the [installation guide](../../install) to install the `osm` cli.
 
-## Install OSM Control Plane
 
-For the purpose of this demo, install OSM with [permissive traffic policy mode](#permissive-traffic-policy-mode) enabled via the `--enable-permissive-traffic-policy` flag. By default, OSM will install with permissive traffic policy mode disabled and [SMI Traffic Policy Mode](#smi-traffic-policy-mode) enabled. Also by default, `osm` CLI does not enable Prometheus, Grafana, and Jaeger as a part of control plane installation.
+## Installing OSM on Kubernetes
 
-Install OSM in permissive traffic policy mode with these features enabled:
+With the `osm` binary downloaded and unzipped, we are ready to install Open Service Mesh on a Kubernetes cluster:
+
+The command below shows how to install OSM on your Kubernetes cluster.
+This command enables
+[Prometheus](https://github.com/prometheus/prometheus),
+[Grafana](https://github.com/grafana/grafana), and
+[Jaeger](https://github.com/jaegertracing/jaeger) integrations.
```

Comment:
Just a quick clarification: the `--deploy-jaeger` flag deploys an instance of Jaeger, but as far as I've understood from @eduser25, OSM doesn't really "integrate" or do anything with it unless `--set OpenServiceMesh.tracing.enable=true` is also used. I am clarifying this is in the tracing doc and manual demo in PR #2971 as well

Reply:
I am going to document this with: https://github.com/openservicemesh/osm/pull/3020